package.json

```diff
@@ -37,36 +38,37 @@
     ]
   },
   "dependencies": {
-    "d3": "^4.13.0",
-    "dagre-d3-renderer": "^0.5.8",
-    "dagre-layout": "^0.8.8",
-    "graphlibrary": "^2.2.0",
-    "he": "^1.1.1",
-    "lodash": "^4.17.5",
-    "moment": "^2.21.0",
-    "scope-css": "^1.0.5"
+    "d3": ">=4.13.0",
+    "dagre-d3-renderer": ">=0.5.8",
+    "dagre-layout": ">=0.8.8",
+    "graphlibrary": ">=2.2.0",
+    "he": ">=1.1.1",
+    "lodash": ">=4.17.5",
+    "moment": ">=2.21.0",
+    "scope-css": ">=1.0.5",
+    "yarn": ">=1.7.0"
   },
   "devDependencies": {
-    "babel-core": "^6.26.0",
-    "babel-loader": "^7.1.4",
-    "babel-preset-env": "^1.6.1",
-    "coveralls": "^3.0.0",
-    "css-loader": "^0.28.11",
-    "css-to-string-loader": "^0.1.3",
-    "gulp": "^3.9.1",
-    "gulp-filelog": "^0.4.1",
-    "gulp-jison": "^1.2.0",
-    "husky": "^0.14.3",
-    "identity-obj-proxy": "^3.0.0",
-    "jest": "^22.4.2",
-    "jison": "^0.4.18",
-    "node-sass": "^4.7.2",
-    "sass-loader": "^6.0.7",
-    "standard": "^11.0.1",
-    "webpack": "^4.1.1",
-    "webpack-cli": "^2.0.12",
-    "webpack-node-externals": "^1.6.0",
-    "yarn-upgrade-all": "^0.3.0"
+    "babel-core": ">=6.26.0",
```

Comment:
Why do you want the devDependencies to be "greater than or equal to" instead of "compatible with"?